src/test/java/org/opentripplanner/_support/time/TestDateTimeUtils.java

```diff
@@ -1,10 +1,10 @@
-package org.opentripplanner.util;
+package org.opentripplanner._support.time;
 
 import java.time.Instant;
 import java.time.ZoneId;
 import java.time.ZonedDateTime;
 
-public class TestUtils {
+public class TestDateTimeUtils {
```

Comment:
I'd just inline these, as they were mostly useful with the old date classes, which had 0-indexed months

Reply:
I inlined the methods, and used the `java.time.Month` instead of the constants.

Reply:
Seems it was changed to `com.google.type.Month.AUGUST_VALUE`, not `java.time.Month.AUGUST`. I think we could just replace the months with numeric values, now that the constructors use 1-based indexing for months, rather than the 0-based i the old `Date` class.

Reply:
`com.google.type.Month` is not used any more. The `LocalDate` types in java take `java.util.Month` as an argument, so I use it with `atZone` instead of using `ZonedDateTime` constructor.

Reply:
BTW, there is also this way to create a LocalDateTime: 

```
LocalDateTime.parse("2022-12-05T12:00");
```